File: 03_21.02_klassikomplekt/kymnev6istlus/src/main/java/ee/tormi/kymnev6istlus/entity/Results.java
package ee.tormi.kymnev6istlus.entity;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Entity

public class Results {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long results_id;

    private Double secondsHundredMeterRun;
    private Double metersLongJump;
    private Double metersShotPut;
    private Double metersHighJump;
    private Double secondsFourHundredMeterRun;
    private Double secondsHundredTenMeterHurdle;
    private Double metersDiscusThrow;
    private Double metersPoleVault;
    private Double metersJavelin;
    private Double secondsThousandFiveHundredMeterRun;
}

File: 13_09.05_kontrolltoo2/FEKT2/src/models/Results.ts
import type { Athlete } from "./Athletes"

export type Results = {
    resultsId: number,
    secondsHundredMeterRun: number,
    metersLongJump: number,
    metersShotPut: number,
    metersHighJump: number,
    secondsFourHundredMeterRun: number,
    secondsHundredTenMeterHurdle: number,
    metersDiscusThrow: number,
    metersPoleVault: number,
    metersJavelin: number,
    secondsThousandFiveHundredMeterRun: number,
    
    athlete: Athlete
}